src/js/geolocation.js

```diff
@@ -0,0 +1,75 @@
+const btnPos = document
+    .querySelector('.leftAsideBox__findCity--button')
+    .addEventListener('click', getLocation);
+const btnWeather = document
+    .querySelector('.leftAsideBox__ChooseCity--button')
+    .addEventListener('click', getWeather);
+const divPos = document.querySelector(".main__box1--header");
+const divWeather = document.querySelector(".main__box1--paragraph");
+
+let inputCity;
+let lat;
+let lon;
+let myRes;
+
+// Get location of navigator
+function getLocation() {
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(showPosition, error);
+  } else { 
+    divPos.innerHTML = "Geolocation is not supported by this browser.";
+  }
+};
+
+async function showPosition(position) {
+  lat = position.coords.latitude;
+  lon = position.coords.longitude;
+  let url = `http://api.openweathermap.org/data/2.5/weather?lat=${lat}&lon=${lon}&APPID=307b855de38a960270e1caa9d305240a`;
+  await fetch(url)
+  .then(function(response) {
+    return response.json();
+  })
```

Comment:
albo async await, albo promise then, prosze sie zdecydowac.

Reply:
Tu mam największy problem bo jak usunę awaita to trzeba dwukrotnie kliknąć button GO żeby wczytał dane. Muszę jeszcze nad tym dzisiaj popracować.